velox/exec/Exchange.cpp

```diff
@@ -14,17 +14,29 @@
  * limitations under the License.
  */
 #include "velox/exec/Exchange.h"
+#include <algorithm>
+#include <random>
 #include "velox/exec/Task.h"
 
 namespace facebook::velox::exec {
 
+void Exchange::addTaskIds(std::vector<std::string> taskIds) {
+  auto rng = std::default_random_engine{};
+  std::shuffle(std::begin(taskIds), std::end(taskIds), rng);
+  for (std::string taskId : taskIds) {
+    exchangeClient_->addRemoteTaskId(taskId);
+    ++stats_.wlock()->numSplits;
```

Comment:
Should we just increase this once after the loop?

  stats_.wlock()->numSplits += taskIds.size();

Reply:
+1